models/commentsModels.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection.js')
+
+exports.selectComments = (reviewId) => {
+    return db.query(
+        `SELECT * FROM comments
+         WHERE review_id = $1
+         ORDER BY created_at DESC`, [reviewId]
+    ).then((res) => {
+        if (!res.rowCount) return Promise.reject({status: 404, msg: "No comments found"})
```

Comment:
We will need to check here the reason for returning an empty row (ie - is it a case of no comments for that valid Id or was it an invalid Id - this will then need to serve the relevant response) - hint is to check whether the review exists :D